packages/ui/src/bounty/hooks/useBounties.ts

```diff
@@ -0,0 +1,28 @@
+import { useGetBountiesCountQuery, useGetBountiesQuery } from '@/bounty/queries'
+import { BountyOrderByInput } from '@/common/api/queries'
+import { usePagination } from '@/common/hooks/usePagination'
+import { SortOrder, toQueryOrderByInput } from '@/common/hooks/useSort'
+
+import { asBounty } from '../types/casts'
+
+interface UseBountiesProps {
+  order: SortOrder<BountyOrderByInput>
+  perPage?: number
+}
+
+export const useBounties = ({ order, perPage = 10 }: UseBountiesProps) => {
+  const orderBy = toQueryOrderByInput<BountyOrderByInput>(order)
+  const { data: dataCount } = useGetBountiesCountQuery()
+  const totalCountPerPage = dataCount?.bountiesConnection.totalCount
+  const { offset, pagination } = usePagination(perPage, totalCountPerPage ?? 0, [order])
+
+  const { loading, data } = useGetBountiesQuery({
```

Comment:
Shouldn't we utilize `where` variable for extra filtering when it's already declared inside schema and it will be needed for [Bounties page](https://www.figma.com/file/jZwIoaxOkTxNFMJNzssBgv/Bounty?node-id=429%3A230948)?

Reply:
I was going to add this in Filters #1942